build.gradle

```diff
@@ -336,7 +359,6 @@ task spec(dependsOn: ['testClasses'], type: JavaExec) {
             "-PS$n", '-T', '120']
     classpath = sourceSets.test.runtimeClasspath
 }
-build.dependsOn spec
```

Comment:
Why remove this?

Reply:
sorry its was my bad. not intentionally so revert this change